src/chains/models.py

```diff
@@ -36,6 +37,9 @@ class Chain(models.Model):
     )
     gas_price_oracle_url = models.URLField(blank=True, null=True)
     gas_price_oracle_parameter = models.CharField(blank=True, null=True, max_length=255)
+    ens_registry_address = models.CharField(
```

Comment:
Use `EthereumAddressField` from `gnosis.eth.django.model` (gnosis-py)

Reply:
Using the `EthereumAddressField` now e3604bc